packages/mock-block-dock/src/bottom-view.tsx

```diff
@@ -0,0 +1,133 @@
+// @todo rename
+
+import { Message } from "@blockprotocol/core/dist/esm/types";
+import { BlockGraphProperties } from "@blockprotocol/graph/.";
+import {
+  KeyboardDoubleArrowDown,
+  KeyboardDoubleArrowUp,
+} from "@mui/icons-material";
+import {
+  Box,
+  IconButton,
+  Paper,
+  PaperProps,
+  styled,
+  Tab,
+  Tabs,
+} from "@mui/material";
+import { Dispatch, SetStateAction, useEffect, useRef, useState } from "react";
+
+import { DataStoreView } from "./bottom-view/datastore-view";
+import { Log, LogsView } from "./bottom-view/logs-view";
+import { PropertiesView } from "./bottom-view/properties";
+import { a11yProps, TabPanel } from "./bottom-view/tab-panel";
+import { SIDEBAR_WIDTH } from "./layout";
+import { MockData } from "./use-mock-block-props/use-mock-datastore";
+
+const Container = styled((props: PaperProps & { minimized: boolean }) => (
+  <Paper {...props} />
+))(({ theme, minimized }) => ({
+  position: "fixed",
+  height: minimized ? 50 : 500,
+  left: SIDEBAR_WIDTH,
+  right: 0,
+  bottom: 0,
+  zIndex: 10,
+  boxShadow: theme.shadows[4],
+  transition: theme.transitions.create("height"),
+  display: "flex",
+  flexDirection: "column",
+}));
+
+const Header = styled(Box)(({ theme }) => ({
+  top: 0,
+  zIndex: 5,
+  display: "flex",
+  justifyContent: "space-between",
+  alignItems: "center",
+  height: 50,
+  borderBottom: `1px solid ${theme.palette.divider}`,
+}));
+
+type BottomViewProps = {
+  graphProperties: BlockGraphProperties<any>;
+  datastore: MockData;
+  readonly: boolean;
+  setReadonly: Dispatch<SetStateAction<boolean>>;
+};
+
+export const BottomView = ({
+  graphProperties,
+  datastore,
+  readonly,
+  setReadonly,
+}: BottomViewProps) => {
+  const [value, setValue] = useState(2);
+  const containerRef = useRef<HTMLDivElement | null>(null);
+  const [logs, setLogs] = useState<Log[]>([]);
+  const [minimized, setMinimized] = useState(false);
+
+  useEffect(() => {
+    const handler = (event: Event) => {
+      const detail = (event as CustomEvent<Message>).detail;
+      setLogs((prev) => [...prev, { ...detail }]);
```

Comment:
This doesn't account for the logs count getting really large. Will address in a follow up